tensorboard/webapp/hparams/_redux/hparams_selectors.ts

```diff
@@ -25,81 +30,126 @@ import {
 const getHparamsState =
   createFeatureSelector<HparamsState>(HPARAMS_FEATURE_KEY);
 
-const getHparamsDefaultFiltersForExperiments = createSelector(
-  getHparamsState,
-  (
-    state: HparamsState,
-    experimentIds: string[]
-  ): Map<string, DiscreteFilter | IntervalFilter> => {
-    const defaultFilterMaps: Array<
-      Map<string, DiscreteFilter | IntervalFilter>
-    > = [];
-
-    for (const experimentId of experimentIds) {
-      if (!state.specs[experimentId]) {
-        continue;
-      }
-
-      defaultFilterMaps.push(state.specs[experimentId].hparam.defaultFilters);
+function getHparamsDefaultFiltersForExperimentsResultFunction(
+  state: HparamsState,
+  experimentIds: string[]
+): Map<string, DiscreteFilter | IntervalFilter> {
+  const defaultFilterMaps: Array<Map<string, DiscreteFilter | IntervalFilter>> =
+    [];
+
+  for (const experimentId of experimentIds) {
+    if (!state.specs[experimentId]) {
+      continue;
     }
 
-    return combineDefaultHparamFilters(defaultFilterMaps);
+    defaultFilterMaps.push(state.specs[experimentId].hparam.defaultFilters);
   }
-);
 
-export const getHparamFilterMap = createSelector(
-  getHparamsDefaultFiltersForExperiments,
+  return combineDefaultHparamFilters(defaultFilterMaps);
+}
+
+const getHparamsDefaultFiltersForExperiments = createSelector(
```

Comment:
So this is basically a selector with a prop right? All current calls use this?

Reply:
Yeah, exactly. It is only used a dependency of other selectors though